examples/async_tasks/async_client.rs

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
These should be module level comments (i.e. `//!`) instead of C-style multiline comments.